static/app/utils/replays/replayReader.tsx

```diff
@@ -70,44 +71,47 @@ export default class ReplayReader {
     replayRecord,
     errors,
   }: RequiredNotNull<ReplayReaderParams>) {
-    const {rawBreadcrumbs, rawRRWebEvents, rawNetworkSpans, rawMemorySpans} =
-      splitAttachmentsByType(attachments);
+    this.replayRecord = replayRecord;
 
-    const spans = [...rawMemorySpans, rawNetworkSpans] as ReplaySpan[];
+    const attachmentsByType = splitAttachmentsByType(attachments);
 
-    // TODO(replays): We should get correct timestamps from the backend instead
-    // of having to fix them up here.
-    const {startTimestampMs, endTimestampMs} = replayTimestamps(
-      replayRecord,
-      rawRRWebEvents as RecordingEvent[],
-      rawBreadcrumbs as ReplayCrumb[],
-      spans
-    );
-    replayRecord.started_at = new Date(startTimestampMs);
-    replayRecord.finished_at = new Date(endTimestampMs);
-    replayRecord.duration = duration(
-      replayRecord.finished_at.getTime() - replayRecord.started_at.getTime()
-    );
+    if (localStorageWrapper.getItem('REPLAY-BACKEND-TIMESTAMPS') !== '1') {
```

Comment:
What is local storage here used for?

Reply:
I want to get rid of this frontend code eventually, and just read data from the backend if we can. So i wanted to create a toggle for testing